docs/reference/esql/esql-limitations.asciidoc

```diff
@@ -64,6 +64,7 @@ include::processing-commands/limit.asciidoc[tag=limitation]
 ** `ip_range`
 ** `long_range`
 ** `nested`
+** `object`
```

Comment:
The story here is complicated. Like, what does "support" mean here. We load fields *in* an `object` field just fine. But we don't support loading the entire `object` field as a tree structure.

Reply:
I guess move it.... I *think* it'd look funky to have a bunch of nuance in this list. Especially around the other fields where the answer is pretty simple - we just can't touch these fields.

Reply:
Yeah I guess the question is what weirdnesses are folks going to hit if they try to work with `object` fields in ES|QL, and are they worth enumerating here at this point in time?

Reply:
It's worth double checking. They don't come back in `*`. If you refer to them directly they are unsupported, but I don't know what that'll look like. But that's not a "normal" thing to do.